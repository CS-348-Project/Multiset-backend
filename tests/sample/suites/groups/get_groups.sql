/*
Name: get_groups.sql
Description: Get all groups
Usage: {}
*/
--prodorder mg.id
SELECT mg.* FROM multiset_group mg;