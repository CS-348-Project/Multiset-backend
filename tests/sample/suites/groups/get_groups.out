[
    {
        "id": 1,
        "name": "Book Club",
        "optimize_payments": true,
        "created_at": "2024-06-20 22:31:50.133020"
    },
    {
        "id": 2,
        "name": "Suite 203",
        "optimize_payments": false,
        "created_at": "2024-06-20 22:31:50.133020"
    },
    {
        "id": 3,
        "name": "Theater Club",
        "optimize_payments": true,
        "created_at": "2024-06-20 22:31:50.133020"
    },
    {
        "id": 4,
        "name": "Pizza Pals",
        "optimize_payments": false,
        "created_at": "2024-06-20 22:31:50.133020"
    },
    {
        "id": 5,
        "name": "The Beatles",
        "optimize_payments": false,
        "created_at": "2024-06-20 22:31:50.133020"
    },
    {
        "id": 6,
        "name": "The Three Stooges",
        "optimize_payments": false,
        "created_at": "2024-06-20 22:31:50.133020"
    },
    {
        "id": 7,
        "name": "Gang of Four",
        "optimize_payments": true,
        "created_at": "2024-06-20 22:31:50.133020"
    }
]